### ISSUE TITLE
Dynamic search control is not keyboard accessible

### ISSUE THREAD

**Issue by mejackreed** [geoblacklight/geoblacklight#269]

**Comment by mejackreed** [comment]
Investigated a bit, and this seems to be an upstream issue with the way Leaflet handles these controls.

**Comment by eliotjordan** [comment]
Need more explanation, maybe an example.

**Comment by mejackreed** [comment]
When tabbing through result/home page, "Search here" is not in tab order

**Comment by eliotjordan** [comment]
Gotcha. Thats what were were thinking was the issue.

**Comment by andrewbattista** [comment]
Tabbing through to the text search box now works in Stanford's Earthworks and in the test GBL app off of the master branch, but not on other public facing sites. I am guessing that it's addressed and we can close this
